mono/mini/mini-runtime.c

```diff
@@ -3700,3 +3737,46 @@ mono_jumptable_get_entry (guint8 *code_ptr)
 	return mono_arch_jumptable_entry_from_code (code_ptr);
 }
 #endif
+
+void
+mono_update_perf_counter (MonoJitTlsData *jit_tls)
```

Comment:
Can't we move all of this code to a separate file? mini and mini-runtime are already pretty big for us to keep adding more stuff to it.